sdcm/cluster.py

```diff
@@ -2803,17 +2803,19 @@ def get_nodetool_status(self, verification_node=None):  # pylint: disable=too-ma
             verification_node = random.choice(self.nodes)
         status = {}
         res = verification_node.run_nodetool('status')
-        data_centers = res.stdout.strip().split("Data center: ")
+        data_centers = res.stdout.strip().split("Datacenter: ")
```

Comment:
how did it work before? Was the `nodetool` string changed recently?

Reply:
> how did it work before?

Work for lucky.
- Invalid result is ignored in counting the living node.
- we only check if some nodes are living, we don't verify which dc the node is in.

> Was the `nodetool` string changed recently?

No, the datacenter string didn't change recently. 
Question of nodetool 5 years ago: https://stackoverflow.com/questions/20678366/how-to-interpret-the-output-of-nodetool-status